### PR TITLE
Generate and update form schemas cli tool

### PR DESCRIPTION
**Pull request by mpinter**
A cli wizard for generating all required files and sending them to correct BE.

We need to know the slugs used on BE somewhere - proposal here is to change to file / directory names of the generated schemas to match those.

### CODE REVIEW

File: next/schema-generator/update.ts [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+  console.log(chalk.blueBright('Generate and update backend form schemas wizard 🧙‍♂️'))
+  const environmentsWithTokens = {
+    'https://nest-forms-backend.staging.bratislava.sk': stagingToken,
+    'https://nest-forms-backend.bratislava.sk': prodToken,
+    'https://nest-forms-backend.dev.bratislava.sk': devToken,
+    'http://localhost:3000': devToken,
+  }
+
+  const environmentResponse = await prompts({
+    onState: (state) => {
```
Comment (MarekBodingerBA) [inline next/schema-generator/update.ts:45]:
```const handleOnStateAborter = (state) => {
      if (state.aborted) {
        process.nextTick(() => {
          process.exit(0)
        })
      }
    }
    
    onState: handleOnStateAborter ```
    ?

File: next/schema-generator/update.ts [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+      headers: {
+        accept: 'application/json',
+      },
+    })
+
+    console.log(
+      `Response status: ${schemaVersionsResponse.status} ${schemaVersionsResponse.statusText}`,
+    )
+
+    const data: any = await schemaVersionsResponse.json()
```
Comment (MarekBodingerBA) [inline next/schema-generator/update.ts:143]: why any?

**Review by MarekBodingerBA — APPROVED** [review]

**Comment by MarekBodingerBA** [comment]
Probably assumes many things are as they are expected (e.g. semver increase, the slug is existing when asking for latest versoin). Also I would use generated API, but LGTM!  🚢

File: next/schema-generator/update.ts (hunk 2 above)
Comment (mpinter) [reply next/schema-generator/update.ts:143]: cause I'm lazy, this is an internal tool and json() returns unknown

Comment (MarekBodingerBA) [reply next/schema-generator/update.ts:143]: ah yea, it doesn't use the api, then it makes sense

File: next/schema-generator/update.ts [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+
+    const id = data?.items?.[0]?.id
```
Comment (MarekBodingerBA) [inline next/schema-generator/update.ts:145]: I would atleast:
```
const firstItem = data?.items?.[0];
if (!firstItem) {
 ...
}
const id = firstItem.id;
....

**Pull request merged**